DataFormats/Reconstruction/include/ReconstructionDataFormats/PID.h

```diff
@@ -33,19 +33,19 @@ namespace o2cp = o2::constants::physics;
 namespace pid_constants // GPUs currently cannot have static constexpr array members
 {
 typedef uint8_t ID;
-static constexpr ID NIDsTot = 17;
+static constexpr ID NIDsTot = 18;
 
 #if !defined(GPUCA_GPUCODE_DEVICE) || defined(GPUCA_GPU_DEBUG_PRINT)
 GPUconstexpr() const char* sNames[NIDsTot + 1] = ///< defined particle names
   {"Electron", "Muon", "Pion", "Kaon", "Proton", "Deuteron", "Triton", "He3", "Alpha",
-   "Pion0", "Photon", "K0", "Lambda", "HyperTriton", "Hyperhydrog4", "XiMinus", "OmegaMinus", nullptr};
+   "Pion0", "Photon", "K0", "Lambda", "HyperTriton", "Hyperhydrog4", "HyperHelium4", "XiMinus", "OmegaMinus", nullptr};
```

Comment:
please add the hyperhe4 after the Omega, it is ugly but we should avoid unforeseen consequences of moving around the ID of Xi and Omegas

Reply:
Hi Max, I have changed the orders of the additional particles: He4L and He5L